Temmietayo/All-Recipes
# #169997323 Template platform for user to delete or modify recipe added by them

#### What does this PR do?
Add the template for user to delete a recipe they added .

#### Description of Task to be completed?
Authenticated User can delete recipe he/she added

#### How should this be manually tested?

- clone the repo

- cd into the template directory

- open the cardview.html with any browser of your choice

- these template should open in that browser
#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[169997323](https://www.pivotaltracker.com/story/show/169997323)
